dolphinscheduler-alert/src/main/java/org/apache/dolphinscheduler/alert/utils/EnterpriseWeChatUtils.java

```diff
@@ -99,9 +99,7 @@ public static String getToken() throws IOException {
                 response.close();
             }
 
-            Map<String, Object> map = JSON.parseObject(resp,
-                    new TypeToken<Map<String, Object>>() {
-                    }.getType());
+            Map<String, Object> map = JSONUtils.parseObject(resp, Map.class);
```

Comment:
parseObject is hard to understand ,suggest add parseMap method ,refer to the following code 
`TypeReference<HashMap<String, String>> typeRef 
  = new TypeReference<HashMap<String, String>>() {};
Map<String, String> map = mapper.readValue(jsonInput, typeRef);`

Reply:
ok，I will fix all similar problems